RELEASES.md

```diff
@@ -63,6 +64,83 @@ Get acquainted with the latest KubeRocketCI releases.
+
+This update also offers new Java 21 language version support. This update comes with corresponding review and build pipelines.
+
+### Upgrades
+
+- Go language is updated to the [1.24](https://tip.golang.org/doc/go1.24) version. ([#444](https://github.com/epam/edp-tekton/issues/444))
+- Align base headlamp image version
+
+### New Functionality
+
+* The deploy pipelines now include a differential approve feature, allowing users to see Argo CD differences in application configuration before approval. Enable it by selecting the diff-approve deploy pipeline in the Environment settings. ([#424](https://github.com/epam/edp-tekton/issues/424)) ask Serdiuk
+* The protected label has been added to the platform. When configured, it denies users to modify or delete a resource within the portal. ([#606](https://github.com/epam/edp-headlamp/issues/606))
+* The `Developer` role now allows to create, modify, and delete Deployment Flow and Environment resources. Additionally, the `Developers` group is now also able to manage `codebasebranch` custom resources. ([#409](https://github.com/epam/edp-install/issues/409))([#400](https://github.com/epam/edp-install/issues/400))
+* KubeRocketCI now supports specifying `imagePullSecrets` for the Tekton service account. This is essential for pulling container images from private registries in Tekton pipelines. ([#427](https://github.com/epam/edp-install/issues/427))
+* External clusters can now be integrated using the IRSA authentication token. ([#640](https://github.com/epam/edp-headlamp/issues/640))
+* Now users can create a private codebase when using the **Create from template** and **Clone project** creation strategies. To create a private codebase, select the **Private** checkbox. ([#676](https://github.com/epam/edp-headlamp/issues/676))
+* Java 21 language version support has been added into the platform. ([#584](https://github.com/epam/edp-headlamp/issues/584))
+* The **Delete Git Server** button has been added to the Version Control System tab of the Configuration section. Now users can delete Git Server using UI instead of running a `kubectl delete` command. ([#654](https://github.com/epam/edp-headlamp/issues/654))
+* Deployment Flows can now be filtered by application name included in them. ([#648](https://github.com/epam/edp-headlamp/issues/648))
+
+
+### Enhancements
+
+* External cluster connection status is now displayed in the Configuration section. ([#686](https://github.com/epam/edp-headlamp/issues/686))
+* A new tab called **kubeconfig** has been added to the **Account settings** window. It allows to view and edit user's `kubeconfig` file. Additionally, this tab also displays the time remaining to session expiration. ([#667](https://github.com/epam/edp-headlamp/issues/667))
```

Comment:
pleasse make more clear statement why we need kubeconfig. It's not about editing or view